doc/manual/judging.rst

```diff
@@ -0,0 +1,168 @@
+Judging topics
```

Comment:
either here or in the advanced config section we should mention lazy judging and results priorities